src/video/sdl_v.h

```diff
@@ -14,8 +14,16 @@
 
 #include "video_driver.hpp"
 
+#ifdef __EMSCRIPTEN__
+void em_loop(void *arg);
```

Comment:
is there a particular reason not to make this a (conditional) class method? would save the duplicate `friend` definition

Reply:
It is called by function pointer by emscripten. Is it safe to assume that calling convention would match?